src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -409,24 +409,23 @@ export default class Pilot {
      *
      * @for Pilot
      * @method cancelApproachClearance
-     * @param currentAltitude {number}  the aircraft's current altitude, in feet
-     * @param currentHeading {number}   the aircraft's current heading, in radians
-     * @return {array}                  [success of operation, readback]
+     * @param aircraftModel {AircraftModel}
+     * @return {array} [success of operation, readback]
      */
-    cancelApproachClearance(currentAltitude, currentHeading) {
+    cancelApproachClearance(aircraftModel) {
         if (!this.hasApproachClearance) {
             return [false, 'we have no approach clearance to cancel!'];
         }
 
         const airport = AirportController.airport_get();
         const altitudeToMaintain = Math.max(
```

Comment:
Could you break this up a bit more?  I'm not a fan of the nested `Math` calls here

Reply:
done!